editor/src/components/canvas/canvas-strategies/strategies/grid-draw-to-insert-strategy.tsx

```diff
@@ -258,6 +259,7 @@ const gridDrawToInsertStrategyInner =
                 : null,
             ]),
           ],
+          'rerender-all-elements',
```

Comment:
possibly one to fix here too

Reply:
Added comment about follow up in 38ec355d5.